src/garage/torch/policies/gaussian_mlp_policy.py

```diff
@@ -0,0 +1,64 @@
+import torch
+from torch import nn
+from garage.torch.modules import GaussianMLPModule
+import akro
+
+class GaussianMLPPolicy(nn.Module):
```

Comment:
This should inherit from `garage.np.Policy`

Reply:
I make new Policy class under the src/garage/torch/policies/base.py
and GaussianMLPPolicy inherit the Policy class